next.config.js

```diff
@@ -5,7 +5,14 @@ module.exports = withContentlayer()({
     optimizeFonts: true,
     modern: true,
   },
-  images: { domains: ['img.youtube.com'] },
+  images: {
+    domains: [
+      'img.youtube.com',
+      'assets-global.website-files.com',
+      'www.brikl.com',
```

Comment:
I would move these images locally, like we do for some of the sponsors. While we're here, can you do that?

Reply:
Sure, I will do that.